src/Microsoft.PowerShell.ConsoleGuiTools/GridViewDataSource.cs

```diff
@@ -31,6 +32,11 @@ public void SetMark(int item, bool value)
         {
             GridViewRowList[item].IsMarked = value;
         }
+        
+        public IList ToList()
```

Comment:
This is part of the `IListDataSource` interface defn. It is not used, and theoretically could through a not impl exception, but I don't see why it can't be implemented.